packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -391,6 +391,15 @@ export const testParasRococoCommon: EndpointOption[] = [
       'Encointer Association': 'wss://rococo.api.encointer.org'
     },
     teleport: [-1]
+  },
+  {
+    info: 'rococoBridgehub',
+    paraId: 1013,
+    text: 'Bridgehub',
+    providers: {
+      Parity: 'wss://rococo-bridge-hub-rpc.polkadot.io'
+    },
+    teleport: [-1]
```

Comment:
Remove the teleport. These are only for system/common chains.

Reply:
Or is it meant to be in that category?

Reply:
It is still under discussion if they plan to implement a teleport into bridge hub parachains. I'll temporarily remove it for now, then add it when there's a decision.